src/js/consts/applicationUrls.js

```diff
@@ -35,6 +35,9 @@ const LOCATION_URL = {
 const STOCK_MOVEMENT_URL = {
   base: '/stockMovement',
   list: () => `${STOCK_MOVEMENT_URL.base}/list`,
+  listInbound: () => `${STOCK_MOVEMENT_URL.base}/list?direction=INBOUND`,
+  listOutbound: () => `${STOCK_MOVEMENT_URL.base}/list?direction=OUTBOUND`,
+  listRequest: () => `${STOCK_MOVEMENT_URL.base}/list?direction=OUTBOUND&sourceType=ELECTRONIC`,
```

Comment:
why can't it be like I've shown? I mean, using the `${STOCK_MOVEMENT_URL.list}/?direction=StockMovementDirection.INBOUND` instead of hardcoding `/list` in every link?

Reply:
`STOCK_MOVEMENT_URL.list` at this point feels unnecessary if we don't use it for `listInbound` and the rest